_pages/username.yaml

```diff
@@ -0,0 +1,5 @@
+ ---
+ githubHandle: Jiahao Huang
+ pageUrl: jiahaoh.github.io/test.GITHUB.IO
```

Comment:
@jiahaoh this page needs to be correct, it does't appear to be working though. I tried going directly to jiahaoh.github.io and it appears to redirect to http://jiahaoh.com, which 404s as well. Perhaps you've set up a [custom domain](https://help.github.com/articles/using-a-custom-domain-with-github-pages/) that needs to be removed?